### ISSUE TITLE
Kinesis Firehose exponential backoff uses initial delay for backoff factor base

### ISSUE THREAD

**Issue by dana-harrington** [akka/alpakka#1045]
The exponential backoff in the Kinesis Firehose support is implemented incorrectly (or at least surprisingly):
https://github.com/akka/alpakka/blob/f3c1e894c8f62ba7c22b6926974592dbde4cb7ca/kinesis/src/main/scala/akka/stream/alpakka/kinesisfirehose/KinesisFirehoseFlowStage.scala#L87
Note that the base of the power is the initial retry delay rather than a constant (typically 2). Contrast with the same code in the Kinesis Streams code with the expected base of 2:
https://github.com/akka/alpakka/blob/f3c1e894c8f62ba7c22b6926974592dbde4cb7ca/kinesis/src/main/scala/akka/stream/alpakka/kinesis/KinesisFlowStage.scala#L87

**Comment by ennru** [comment]
Agreed, that's surprising. Would you be able to send a Pull Request to change this?

**Comment by dana-harrington** [comment]
Of course, [the PR is here.](https://github.com/akka/alpakka/pull/1048)
